### ISSUE TITLE
Add exact instructions on how to install ndn-cxx dependency

### ISSUE THREAD

**Issue by neel1998** [named-data-mobile/ndn-photo-app#175]
**Summary:** 
Currently README lacks the link to ndn-cxx repo and how to install this library and its dependecny

**Would you like to work on the issue?**
Yes

@jrandallclark adding exact steps to readme would help the new contributors a lot, Since i myself had to struggle a bit

**Comment by jrandallclark** [comment]
An npChat developer doesn't need to install ndn-cxx. I'm not sure what the purpose of this is. Could you explain?

**Comment by neel1998** [comment]
As it is mentioned in the setting up guide of this repo, also doesn't the psync setup require to have ndn-cxx setup. Doesn't Android build fail without setting up psync and ndn-cxx?

**Comment by jrandallclark** [comment]
Oh, I think I understand. The building instructions might be unclear then. ndn-cxx is the C++ library for NDN. You need it installed on your system to build/run any NDN programs on your PC.

PSync Android and NFD Android are both C++ programs with a JNI wrapper. So you just need to use the crew staging to download the ndn-cxx (and other dependencies) binaries. When PSync is built with gradle, it packs those binaries with it. However, you're not installing ndn-cxx on your system.

The installation process for PSync has been simplified from it's initial release. Maybe the instructions should be rewritten to remove the word "install" and to specify that the crew staging is only downloading pre-compiled binaries.

**Comment by neel1998** [comment]
Well then I might have spent a lot time fixing the CMake and build errors by self and installing all the ndn-cxx dependency manually. xD. However, it would be good if you could update the readme so other developers don't get confuse.

**Comment by jrandallclark** [comment]
Well, if you ever want to write something on your system that interacts with npChat, you'll need ndn-cxx and NFD anyways. So maybe it was all worth it. 

I'll go through the PSync build process myself this week and see how I can simplify the instructions.

**Comment by neel1998** [comment]
> Well, if you ever want to write something on your system that interacts with npChat, you'll need ndn-cxx and NFD anyways. So maybe it was all worth it.
> 
Yes, indeed and I also enjoy tweaking with software and OS to find a way to make everything work. So it was fun.

> I'll go through the PSync build process myself this week and see how I can simplify the instructions.

Awesome. Let me know if I can help/
